src/domain/synthetics/trade/utils/swap.ts

```diff
@@ -242,3 +242,23 @@ export function getSwapAmountsByToValue(p: {
     swapPathStats,
   };
 }
+
```

Comment:
I mean, maybe we can break it down into two types-- 

sortByLiquidity and sortByPath?

To not confuse with ternar operators